static/app/api.tsx

```diff
@@ -243,7 +243,10 @@ type ClientOptions = {
    * The base URL path to prepend to API request URIs.
    */
   baseUrl?: string;
-
+  /**
+   * Credentials policy to apply to each request
+   */
+  credentials?: RequestCredentials;
```

Comment:
Does this equate to the `Access-Control-Allow-Credentials` HTTP response header?

If so, what is the `Access-Control-Allow-Origin` header set to? Are we going to be allowing credentials to be sent cross-origin?

Reply:
Yes. according to the current code, we are enabling it to be sent cross-origin and in the other PR we need to pass 'omit'. 

As Mark mentioned below, `include` is the least 'safe' value